ask-sdk-model/ask_sdk_model/services/proactive_events/proactive_events_service_client.py

```diff
@@ -25,7 +25,7 @@
 from ask_sdk_model.services.api_response import ApiResponse
 
 from ask_sdk_model.services.authentication_configuration import AuthenticationConfiguration
```

Comment:
These imports and the other service imports in the class needs to be changed. But won't these be updated during the model generation? Similarly for `skill_messaging_service_client.py`.

Reply:
Yes, will update them for now, yes they will be updated during the model generation.